.github/dependabot.yaml

```diff
@@ -1,15 +1,22 @@
 # dependabot.yml reference: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
-# Automatically update python packages for the jupyterhub/k8s-hub image
-- package-ecosystem: pip
-  directory: "/images/hub"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 4
-  versioning-strategy: lockfile-only
-  labels:
-    - maintenance
-    - dependencies
-  commit-message:
-    prefix: "hub image: "
+  # Maintain Python dependencies for the jupyterhub/k8s-hub image
+  - package-ecosystem: pip
+    directory: "/images/hub"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 4
+    versioning-strategy: lockfile-only
+    labels:
+      - maintenance
+      - dependencies
```

Comment:
This is overriding the default dependabot labels:
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/customizing-dependency-updates#setting-custom-labels

Are we using them for a particular purpose, or could we remove this bit of the config and rely on the defaults?

Reply:
Interesting to know! Hmm want github-activity to sort these PR quickly when generating a changelog, which based on current github-activity function means the maintenance label.

Id like a dedicated category for dependencies etc, but at this point its maintenance.

Can we add extra labels in a hardcoded way instead?

Reply:
Sure, that's fine 😀. Just wanted to check it's an intentional override.

Reply:
:) thanks for review @manics! Merge?